genesis/genesis.go

```diff
@@ -420,55 +394,46 @@ func FromConfig(config *Config) ([]byte, ids.ID, error) {
 			return nil, ids.Empty, err
 		}
 
-		utxos := []api.UTXO(nil)
+		allocations := []api.Allocation(nil)
 		for _, allocation := range nodeAllocations {
 			addr, err := address.FormatBech32(hrp, allocation.AVAXAddr.Bytes())
 			if err != nil {
 				return nil, ids.Empty, err
 			}
 			for _, unlock := range allocation.UnlockSchedule {
-				msgStr, err := formatting.Encode(defaultEncoding, allocation.ETHAddr.Bytes())
```

Comment:
why is this removed?

Reply:
`defaultEncoding` is no longer needed for this functions. So no we no longer need to encode it the bytes. Same for above.